### PR TITLE
4425 update expansion panels

### PR DESCRIPTION
**Pull request by Liliya-web**

### CODE REVIEW

File: components/carousels/VerticalCarousel.vue [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<template>
```
Comment (omega790) [inline components/carousels/VerticalCarousel.vue:1]: Please delete changes related carousels from this PR

File: components/expansion_panels/TileExpansionPanel.vue [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<template>
+  <v-row justify="center">
+    <v-expansion-panels id="TileExpansionPanels" tile>
```
Comment (omega790) [inline components/expansion_panels/TileExpansionPanel.vue:3]: Please use id in one place for this element

File: components/carousels/VerticalCarousel.vue (hunk 1 above)
Comment (Liliya-web) [reply components/carousels/VerticalCarousel.vue]: Done

File: components/expansion_panels/TileExpansionPanel.vue (hunk 2 above)
Comment (Liliya-web) [reply components/expansion_panels/TileExpansionPanel.vue]: Done

**Review by omega790 — APPROVED** [review]

**Pull request merged**